internal/host/transition.go

```diff
@@ -512,7 +512,8 @@ func (th *transitionHandler) PostRefreshHost(reason string) stateswitch.PostTran
 		if err != nil {
 			return err
 		}
-		if sHost.host.Progress.CurrentStage == models.HostStageWritingImageToDisk {
```

Comment:
The statusInfoInstallationInProgressWritingImageToDiskTimedOut message should only be set if the transition that caused it is the "timeout" transition.

The issue was that when the cluster entered an error state, a special refresh transition (who's purpose is to move all hosts into the error state as well) entered this `if` statement although it shouldn't have.

This made it look like hosts that were in the middle of writing to disk while the cluster entered an error state were timing out - even though what actually happened is they simply entered the error state like the rest of the hosts in the cluster.